systems/controllers/controlUtil.h

```diff
@@ -40,6 +40,13 @@ typedef struct _support_state_element
   bool support_logic_map[4];
 } SupportStateElement;
 
+struct DrakeRobotState {

```

Comment:
partly. It also just seemed like a good abstraction: there were a ton of functions that all took (t, q, qd) in some order, and this made it easy to be sure I always got the correct variables in the correct order.